src/loki/isochrone_action.cc

```diff
@@ -47,27 +47,19 @@ void loki_worker_t::init_isochrones(valhalla_request_t& request) {
     l.clear_heading();
   }
 
-  // make sure the isoline definitions are valid
-  auto contours =
-      rapidjson::get_optional<rapidjson::Value::ConstArray>(request.document, "/contours");
-  if (!contours) {
-    throw valhalla_exception_t{113};
-  };
   // check that the number of contours is ok
-  if (contours->Size() > max_contours) {
+  if (request.options.contours_size() < 1) {
```

Comment:
why not throw these in the parsing function, is it because max_contours is a config value we dont have access to? if so, we should fix that and do all the stuff in all one place eventually. for this round no worries though